src/main/java/cn/nukkit/Server.java

```diff
@@ -216,6 +217,8 @@
 
     private final Set<String> ignoredPackets = new HashSet<>();
 
+    private static final Pattern UUID_PATTERN = Pattern.compile("^[0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12}.dat$");
```

Comment:
would make more sense to only use `[0-9a-f]` and compile it with `Pattern.CASE_INSENSITIVE`